static/app/views/projectDetail/charts/projectBaseSessionsChart.tsx

```diff
@@ -62,13 +65,19 @@ function ProjectBaseSessionsChart({
   const {projects, environments, datetime} = selection;
   const {start, end, period, utc} = datetime;
 
+  const Request = [DisplayModes.ANR_RATE, DisplayModes.FOREGROUND_ANR_RATE].includes(
+    displayMode
+  )
+    ? ProjectSessionsAnrRequest
+    : ProjectSessionsChartRequest;
```

Comment:
I found that I was getting errors if I had `SENTRY_RELEASE_HEALTH = "sentry.release_health.sessions.SessionsReleaseHealthBackend"` since `anr_rate()` and `foreground_anr_rate()` weren't being returned. Should we also be checking that condition here or can we assume it's going to be using the metrics backend?